CMakePresets.json

```diff
@@ -6,7 +6,7 @@
       "name": "default",
       "displayName": "Automatic options (debug with tests)",
       "description": "Dependencies are enabled based on environment probing",
-      "binaryDir": "${sourceDir}/build",
+      "binaryDir": "${sourceDir}/build-${presetName}",
```

Comment:
think you might also need to update the build dir in run-ci.sh

Reply:
Ah hm I need to manually add that to the ci presets then. Thanks.